roles/docker/templates/daemon.json

```diff
@@ -14,6 +14,9 @@
 	    "path": "/usr/libexec/docker/docker-runc-current"
 	}
     },
+{% if openshift.node is defined and openshift.node.sdn_mtu is defined %}
+    "mtu": {{ openshift.node.sdn_mtu }},
```

Comment:
This option can't live in both ```/etc/sysconfig/docker-network``` and ```/etc/docker/daemon.json``` as it will cause docker to fail to start up (even if the options are exactly the same.

Note that container-engine honors ```/etc/sysconfig/docker-network``` via it's service file:

```
[Service]
EnvironmentFile=-/etc/sysconfig/docker-storage
EnvironmentFile=-/etc/sysconfig/docker-network
```